src/CountingWords.ts

```diff
@@ -1,13 +1,19 @@
 const countingWords = (phrase: string) => {
   let count = 0
 
-  const wordsArray = phrase.split(' ')
+  try {
+    const wordsArray = phrase.split(' ').filter(word => word != '')
+    
+    if (wordsArray != []) {
```

Comment:
Creo que puede quedar más claro algo así:

```suggestion
    if (wordsArray.length()) {
```

De todos modos "realmente necesitamos comprobar si el array está vacío?" 🤔